charts/seed-bootstrap/charts/vpa/charts/application/templates/crd-verticalpodautoscalers.yaml

```diff
@@ -96,512 +96,100 @@ spec:
     singular: verticalpodautoscaler
   scope: Namespaced
   versions:
-  - additionalPrinterColumns:
-    - jsonPath: .spec.updatePolicy.updateMode
-      name: Mode
-      type: string
-    - jsonPath: .status.recommendation.containerRecommendations[0].target.cpu
-      name: CPU
-      type: string
-    - jsonPath: .status.recommendation.containerRecommendations[0].target.memory
-      name: Mem
-      type: string
-    - jsonPath: .status.conditions[?(@.type=='RecommendationProvided')].status
-      name: Provided
-      type: string
-    - jsonPath: .metadata.creationTimestamp
-      name: Age
-      type: date
-    name: v1
+  - name: v1
```

Comment:
I guess the comment in https://github.com/gardener/gardener/blob/9257f19a9dbded0db9a63b259c3acb0b02b9d4da/charts/seed-bootstrap/charts/vpa/charts/application/templates/crd-verticalpodautoscalers.yaml#L74-L78 has to be adapted that we now set `x-kubernetes-preserve-unknown-fields: true` for compatibility reasons.

Reply:
Sorry @ialidzhikov, I forgot to force push the comment which I amended after opening this PR. I hope the new comment is fine for you?